cmd/batchDataExporter/README.md

```diff
@@ -0,0 +1,25 @@
+| `BUCKETS_RATE_LIMIT`              | _3_                                      | _Specifies maximum number of buckets that can be synced in parallel._                                                               |
+| `DATA_ENCRYPTION_KEY`             | _none_, **_required_**                   | _Base64-encoded data encryption key for sanitizer._                                                                                 |  |
+| `SANITIZER_CONFIG_FILEPATH`       | _/sanitizerConfig.json_                  | _*Path to JSON file with configuration of fields to sanitize for data sanitizer.*_                                                  |
+| `BOLT_DB_FILEPATH`                | `/data/batchDataExporter.db`             | _Path to Bolt DB file in which command saves datetime of last succesful run._                                                       |
+| `STORAGE_HOST`                    | `cloudStorage`                           | _Hostname of source Storage API, used as source storage for sync._                                                                  |
+| `STORAGE_PATH`                    | `storage`                                | _Root path of source Storage API, used as source storage for sync._                                                                 |  |  |
+| `PROMETHEUS_PUSH_GATEWAY_ADDRESS` | `http://localPrometheusPushGateway:9091` | _Full address of Prometheus Push Gateway to push metrics from a single run of the command._                                         |
+| `DB_USERNAME`                     | _none_, **_required_**                   | _PostgreSQL DB username._                                                                                                           |
+| `DB_PASSWORD`                     | _none_, **_required_**                   | _PostgreSQL DB password._                                                                                                           |
+| `POSTGRES_HOST`                   | `postgres`                               | _Hostname on which postgres is exposed on._                                                                                         |
+| `POSTGRES_DATABASE`               | `reports`                                | _Postgres database to connect to._                                                                                                  |
+| `POSTGRES_ROLE`                   | `dataexportservice`                      | _Postgres role to assume once connected._                                                                                           |
+| `DB_DETAILED_LOG`                 | `false`                                  | _Allows to enable detailed DB statements log, otherwise only errors are printed._                                                   |
```

Comment:
config file should also be described here

Reply:
will add